pkg/consts/konnect.go

```diff
@@ -11,4 +15,17 @@ const (
 	// properly referenced by KongPluginBindings.
 	// It avoids that KongPlugins get deleted when KongPluginBindings are still referencing them.
 	PluginInUseFinalizer = "gateway.konghq.com/plugin-in-use"
+	// DataPlaneKonnectExtensionFinalizer is the finalizer added to the secret
+	// referenced by KonnectExtension to ensure that the secret is not deleted
+	// when in use by an active KonnectExtension.
+	SecretKonnectExtensionFinalizer = "gateway.konghq.com/secret-in-use"
+)
+
+// -----------------------------------------------------------------------------
+// Consts - Labels
+// -----------------------------------------------------------------------------
+const (
+	// DataPlaneCertificateIDLabelKey is the label key used to store the certificate ID
```

Comment:
Can we elaborate a bit more on the `certificate ID` here? This is the Konnect ID right? If so, I'd suggest to mention it in the comment explicitly.